apps/src/templates/CensusForm.jsx

```diff
@@ -90,6 +112,7 @@ class CensusForm extends Component {
         <div style={styles.options}>
           {CSOptions.map((label, index) =>
             <Checkbox
+              name={`${label}_b`}
```

Comment:
This isn't right -- CSOptions are different in every language, but you want the `name` here to be the same in every language. You probably need another data structure which stores the canonical name for these. Please feel free to ping me if you'd like more specific guidance on how to accomplish this. 

Alternately, since the CSOptions aren't making it into the forms table yet, you could also just remove this line from the PR for now and tackle it in the next PR.